spec/unit/application/master_spec.rb

```diff
@@ -338,6 +338,27 @@ def a_user_type_for(username)
         end
       end
 
+      it "should log a deprecation notice when running a WEBrick server" do
+        Puppet.features.stubs(:root?).returns true
+        a_user_type_for("puppet").expects(:exists?).returns true
+        Puppet::Util.expects(:chuser)
+
+        Puppet.expects(:deprecation_warning).with("The WEBrick Puppet master server is deprecated and will be removed in a future release. Please use Puppet Server instead. See http://links.puppetlabs.com/deprecate-rack-webrick-servers for more information.")
+
+        @master.main
+      end
+
+      it "should log a deprecation notice when running a Rack server" do
```

Comment:
@whopper Could you restrict this test with `:if => Puppet.features.rack?`, otherwise, if you do `bundle install --without development extra`, we don't install rack gem, and we get a failed require:

```
1) Puppet::Application::Master when running the main command should log a deprecation notice when running a Rack server
     Failure/Error: @master.main
     LoadError:
       cannot load such file -- rack
     # ./lib/puppet/network/http/rack.rb:1:in `require'
     # ./lib/puppet/network/http/rack.rb:1:in `<top (required)>'
     # ./lib/puppet/application/master.rb:302:in `require'
     # ./lib/puppet/application/master.rb:302:in `start_rack_master'
     # ./lib/puppet/application/master.rb:202:in `main'
     # ./spec/unit/application/master_spec.rb:359:in `block (4 levels) in <top (required)>'
```